api/src/main/java/com/velocitypowered/api/proxy/messages/MinecraftChannelIdentifier.java

```diff
@@ -52,9 +52,10 @@ public static MinecraftChannelIdentifier create(String namespace, String name) {
     Preconditions.checkArgument(!Strings.isNullOrEmpty(namespace), "namespace is null or empty");
     Preconditions.checkArgument(name != null, "namespace is null or empty");
     Preconditions.checkArgument(VALID_IDENTIFIER_REGEX.matcher(namespace).matches(),
-        "namespace is not valid");
+        String.format("namespace is not valid, must match: %s got %s", VALID_IDENTIFIER_REGEX.toString(), namespace));
```

Comment:
Instead of formatting manually on every check, use the `checkArgument` methods that take a message template, see https://guava.dev/releases/20.0/api/docs/com/google/common/base/Preconditions.html#checkArgument-boolean-java.lang.String-java.lang.Object-java.lang.Object-

Reply:
Fixed the issue, I removed String.format